ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationContent.tsx

```diff
@@ -28,10 +26,10 @@ export type NavigationContentProps = {
 
 function NavigationContent({ children, variant }: NavigationContentProps): ReactElement {
     return (
-        <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+        <>
             <span>{children}</span>
-            <span>{badgeMap[variant]}</span>
-        </Flex>
+            <span className="pf-v5-u-ml-sm">{badgeMap[variant]}</span>
+        </>
     );
 }
```

Comment:
This updates the layout to ensure the text and layout are rendered inline with each other. Before this change, the label would be pushed down to a new line below the navigation text.